src/main/java/com/treasure/hunt/view/MainController.java

```diff
@@ -29,6 +30,7 @@
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
 
+import java.awt.*;
```

Comment:
* imports are dangerous and awt imports are always suspicious. What are u using it for?